selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -57,6 +59,17 @@ def __init__(self, CP, init_v=0.0, init_a=0.0):
     self.a_desired_trajectory = np.zeros(CONTROL_N)
     self.j_desired_trajectory = np.zeros(CONTROL_N)
     self.solverExecutionTime = 0.0
+    self.params = Params()
+    self.param_read_counter = 0
+    self.read_param()
+    self.personality = log.LongitudinalPersonality.standard
```

Comment:
I know this PR is several months old and all, but back during https://github.com/commaai/openpilot/pull/24742/commits/12321fe544311c7fd1a44baf39ba02466e4ad91d it was necessary to initialize `self.personality` in case `read_params` didn't.

However, in the final version `read_param` always writes to `self.personality` so this line is now counter-productive right?